src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+    Rename and move around final output files for upload
+    '''
+    mark-section "Formatting output files for upload"
+
+    vcf_dir=out/result_files/CNV_vcfs
+    summary_dir=out/result_files/CNV_summary
+    vis_dir=out/result_files/CNV_visualisation
+
+    mkdir -p $vcf_dir $summary_dir $vis_dir
+
+    mv inputs/vcfs/*.vcf ${vcf_dir}/
+    mv excluded_intervals.bed ${summary_dir}/$run_name"_excluded_intervals.bed"
```

Shellcheck

[warning] run_name is referenced but not assigned.

(SC2154)